src/tests/ftest/harness/pre_launch.py

```diff
@@ -0,0 +1,41 @@
+"""
+(C) Copyright 2021-2022 Intel Corporation.
+
+SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import re
+
+from apricot import TestWithoutServers
+
+from run_utils import run_remote
+
+
+class HarnessPreLaunchTest(TestWithoutServers):
+    """Harness pre-launch test cases.
+
+    :avocado: recursive
+    """
+
+    def test_harness_pre_launch_users(self):
+        """Verify all expected users are setup correctly.
+
+        :avocado: tags=all
+        :avocado: tags=vm
+        :avocado: tags=harness
+        :avocado: tags=test_harness_pre_launch_users
```

Comment:
Just for my benefit, when will this test get run?

Reply:
Can I ask that you make this into a function so that when we have tests that need this setup they can call the function to verify it's there and will fail correctly if it's not.

Reply:
The users are now created by launch.py, so they will exist when the tests run

Reply:
Also added functions in user_utils.py that can be used in test cases to sanity check existence